cmd/system-probe/api/server.go

```diff
@@ -45,6 +46,7 @@ func StartServer(cfg *config.Config) error {
 	mux.HandleFunc("/module-restart/{module-name}", restartModuleHandler).Methods("POST")
 
 	mux.Handle("/debug/vars", http.DefaultServeMux)
+	mux.Handle("/telemetry", telemetry.Handler())
```

Comment:
what about the `/telemetry` that is initialized [here](https://github.com/DataDog/datadog-agent/blob/main/cmd/system-probe/subcommands/run/command.go#L214) ?

Reply:
That is on the default mux, which is exposed only on HTTP.

Reply:
talked about that